mava/logger.py

```diff
@@ -27,7 +27,7 @@
 from mava.utils.logger_tools import Logger, get_experiment_path, should_log
 
 
-def get_logger_tools(logger: Logger, config: Dict) -> Tuple[Callable, Callable]:  # noqa: CCR001
+def get_logger_tools(logger: Logger, config: Dict) -> Callable:  # noqa: CCR001
```

Comment:
You can remove `# noqa: CCR001` as this mypy error won't exist after these changes

Reply:
Seems like it's still failing for me if I remove this :cry:

Reply:
yeah I recheck it and it doesn't work unfortunately :(